core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -6348,6 +6409,62 @@ components:
             Caller must have the add_project action for this service perimeter in
             Sam.
       description: ""
+    SpendReport:
+      required:
+        - spendDetails
+        - spendSummary
+      type: object
+      properties:
+        spendDetails:
+          description: details of spend data, categorized by given key
+          type: array
+          items:
+            $ref: '#/components/schemas/SpendReportAggregation'
+        spendSummary:
+          $ref: '#/components/schemas/SpendReportForDateRange'
+    SpendReportAggregation:
+      required:
+        - aggregationKey
+        - spendData
+      type: object
+      properties:
+        aggregationKey:
+          type: object
+          description: Key indicating how spendData has been aggregated. Ex. 'workspace' if all data in spendData is for a particular workspace
```

Comment:
Will all the supported aggregation keys be documented?

Reply:
I was thinking that supported aggregation keys would be documented by the enum when we add that in. Swagger supports enums, so I'll update the swagger to reflect the supported aggregation keys as well